content/applications/finance/accounting/fiscal_localizations/localizations/france.rst

```diff
@@ -2,24 +2,343 @@
 France
 ======
 
-FEC 
-===
+.. _france/fec:
 
-If you have installed the French Accounting, you will be able to download the FEC.
-For this, go in :menuselection:`Accounting --> Reporting --> France --> FEC`. 
+FEC - Fichier des Écritures Comptables
+======================================
+ 
+An :dfn:`Fichier des Écritures Comptables` audit file contains all the accounting data and entries
```

Comment:
Whoops. Mended.

But can I ask why "An FEC audit file" instead of "A FEC audit file"?
"A Fichier des Écritures Comptables" sounds much better to me, but maybe I'm wrong and I'm ignoring some rule 🤔 

![immagine](https://user-images.githubusercontent.com/1665365/146446915-44955a5d-a27c-49dc-b2f0-502328c27f3b.png)

Reply:
@jcs-odoo will probably explain that better than me. In the meantime, it's r+'ed!

Reply:
A and An actually depends on the sound of the sentence, and not the main word.

So indeed, it's A file. But AN "èff-euh-cé" (FEC) file.

Weird, I know ^^

@lordkrandel @AntoineVDV

Reply:
So indeed "A fichier des écritures comptables", and "AN FEC" because of the letters said out loud.